src/flex/agentApp/src/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the MIT License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+import * as FlexPlugin from 'flex-plugin';
+import WatsonAssistantAgentAppPlugin from './WatsonAssistantAgentAppPlugin';
+FlexPlugin.loadPlugin(WatsonAssistantAgentAppPlugin);
```

Comment:
Do you have eslint enabled for this project? There's a linter error here.

Reply:
I think I missed this with earlier changes but I think we should avoid using `index.ts` files unless they're necessary for some reason. We should have dependencies that have explicit names instead of relying on picking up the default file in a folder. Can we give this file a real name?

Reply:
sure, a lot of the code here is mainly from the Twilio Plugin code generator, I'll fix up the code so it's more consistent

Reply:
The Twilio CLI for developing/deploying this does expect an index.ts file to exist in the `src/` directory, so this is definitely an exception